src/hdr_histogram.c

```diff
@@ -21,8 +21,19 @@
 #define HDR_MALLOC_INCLUDE "hdr_malloc.h"
 #endif
 
+/* when using for YB purposes, we set subbucket size manually rather than with sig figs value */
+#define YB_SIG_FIGS 1
```

Comment:
Shouldn't such config value be passed as parameters from the caller so that it is "configurable" without recompiling the whole package?
How likely this will be changed? If not frequently, I am OK to leave it here but we may need a `TODO`.

Reply:
Very unlikely - memory consumption is unrealistic for higher sig fig values. Will add a note.